core/templates/pages/exploration-editor-page/services/angular-name.service.ts

```diff
@@ -23,8 +23,17 @@ import { downgradeInjectable } from '@angular/upgrade/static';
   providedIn: 'root'
 })
 export class AngularNameService {
+  /**
+   * Static type variable to store angular names.
```

Comment:
Static type variable to store angular name.

Reply:
Updated